portfolio/src/main/webapp/css/master.css

```
@@ -0,0 +1,9 @@
+.project-card {
+  transition: 0.1s;
+}
+
+.project-card:hover {
+  -webkit-box-shadow: 2px 2px 5px 1px rgba(232,232,232,1);
+  -moz-box-shadow: 2px 2px 5px 1px rgba(232,232,232,1);
+  box-shadow: 0px 0px 5px 1px rgba(232,232,232,1);
```

Alphabetize these. https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order
```suggestion
  box-shadow: 0px 0px 5px 1px rgba(232,232,232,1);
  -moz-box-shadow: 2px 2px 5px 1px rgba(232,232,232,1);
  -webkit-box-shadow: 2px 2px 5px 1px rgba(232,2
```

---

Oh wow. I definitely should've been referencing the HTML/CSS style guide before this. Thanks